application.go

```diff
@@ -272,11 +351,45 @@ func (r *Application) CheckTCP(port, interval int) (*Application, error) {
 	health.IntervalSeconds = interval
 	health.PortIndex = portIndex
 	// step: add to the checks
-	r.HealthChecks = append(r.HealthChecks, health)
+	r.AddHealthCheck(*health)
 
 	return r, nil
 }
 
+// AddUris adds one or more uris to the applications
+//		arguments:	the uri(s) you are adding
+func (r *Application) AddUris(newUris ...string) *Application {
```

Comment:
I wouldn't lint everything now in order to keep the PR focused and drive towards shipping, but making a separate linting PR (or multiple ones to minimize breakage) sounds good to me.

So in favor of consistent names for now. :+1: